libs/wire-api/src/Wire/API/Team/Feature.hs

```diff
@@ -517,7 +517,7 @@ instance ToSchema ValidateSAMLEmailsConfig where
   schema = object "ValidateSAMLEmailsConfig" objectSchema
 
 instance IsFeatureConfig ValidateSAMLEmailsConfig where
-  type FeatureSymbol ValidateSAMLEmailsConfig = "validateSAMLemails"
+  type FeatureSymbol ValidateSAMLEmailsConfig = "validateSAMLEmails"
```

Comment:
This could be a breaking change because URL should be case-sensitive, however in practice they might not be.